pf/internal/convert/defaults.go

```diff
@@ -0,0 +1,23 @@
+package convert
+
+import (
+	"os"
+
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+)
+
+func GetDefaultValue(d *tfbridge.DefaultInfo, resource *tfbridge.PulumiResource) (interface{}, error) {
```

Comment:
I'd not place it in convert package as that package is dedicated to supporting conversions between PropertyValue and tftypes.Value, and this functionality does not touch tftypes.Value.

Reply:
Is there a place you would like it to be?

Reply:
I'd just throw it in a new file in top-level (under `pf/*`), like `pf/defaults.go` we can start with that and reorg later.

Reply:
Moved the code